### PR TITLE
fix(sysrc-svc): workaround *BSD minion indefinitely hanging on start 

### PR DESCRIPTION
**Pull request by sticky-note**
Check https://github.com/saltstack/salt/issues/44848 for further details

### CODE REVIEW

**Comment by alxwr** [comment]
@vutny If you don't have any FreeBSD hosts to test this PR: I've got them and I use this formula. :-)
@sticky-note Thanks for this!

**Comment by sticky-note** [comment]
@alxwr Is this the correct way to do this ? I've seen you changed the way to do this on https://github.com/saltstack-formulas/prometheus-formula but don't know if it is applicable here

**Comment by alxwr** [comment]
@sticky-note I still use `>/dev/null 2>&1`. Just refactored my code a bit.

File: postgres/server/init.sls [hunk 1]
```diff
@@ -261,12 +261,21 @@ postgresql-tablespace-dir-{{ name }}-fcontext:
 
 {%- if not postgres.bake_image %}
 
+# Workaround for FreeBSD minion undefinitely hanging on service start
+# cf. https://github.com/saltstack/salt/issues/44848
+{% if postgres.service.sysrc %}
+posgresql-rc-flags:
+  sysrc.managed:
+    - name: {{ postgres.service.name }}_flags
+    - value: "{{ postgres.service.flags }} > /dev/null 2>&1"
```
Comment (alxwr) [inline postgres/server/init.sls]: @sticky-note Please apply this patch here:

```diff
diff --git a/postgres/server/init.sls b/postgres/server/init.sls
index 21b33a6..ce1d575 100644
--- a/postgres/server/init.sls
+++ b/postgres/server/init.sls
@@ -268,6 +268,8 @@ posgresql-rc-flags:
   sysrc.managed:
     - name: {{ postgres.service.name }}_flags
     - value: "{{ postgres.service.flags }} > /dev/null 2>&1"
+    - watch_in:
+      - service: postgresql-running
 {% endif %}
 
 # Start PostgreSQL server using OS init
```

Comment (sticky-note) [reply postgres/server/init.sls]: Oh yes, you're right. Well seen

**Comment by sticky-note** [comment]
@alxwr Updated

**Comment by myii** [comment]
@sticky-note We've got `semantic-release` active on this repo (it looks like you already know this).  The commit message is just failing the `commitlint` because of the capital letter.  If you can amend the commit and then force push it back again, it will ensure you get a release once this PR is merged:

```diff
-fix(sysrc-svc): Workaround *BSD minion undefinitely hanging on start
+fix(sysrc-svc): workaround *BSD minion undefinitely hanging on start
```

**Comment by myii** [comment]
@alxwr @vutny Thanks for the reviews.
@sticky-note Thanks for the contribution -- merged.

**Pull request merged**